### ISSUE TITLE
Implement a membership scheme

### ISSUE THREAD

**Issue by goldfirere** [haskellfoundation/tech-proposals#28]
Disclaimer: I don't know exactly where to put this idea, but this seems at least adjacent to the issues in this repo. Anyone with a better idea of where to put this is free to move it.

A recent conversation about Haskell (https://discourse.haskell.org/t/i-feel-like-haskell-is-dying/4180/78) mentioned the idea of HF "membership"; a participant said they felt there was a "high bar of entry".

There are at least two problems already: the HF does not have a well-defined notion of membership (beyond, say, being part of an official committee), and we are perceived to have a high bar of entry. Of course, if the only avenue toward membership is committee work, then that is a high bar: there needs to be a committee of interest with an opening, and the selection process can be seen as competitive. (For board membership, for example, it truly *is* competitive.) But what if someone just wants to be part of the group promoting Haskell and its interests? That person should have a way of declaring their own affiliation with the HF. I don't yet have a concrete idea for what this affiliation would entail -- that's why this is an Issue, not a PR -- but I do see such a thing as useful. In a sense, this could create "card-carrying Haskellers". Maybe we let anyone declare themselves a "member" and give out cards to people who, say, contribute at least $5/month? And then call on members for volunteer opportunities? Not sure.

My bottom line here: I think there are a decent number of people who care about Haskell and its future out there. Let them declare this (publicly) and be counted, and that can only be good for the language and the HF.

**Comment by simonpj** [comment]
Yes -- good idea!

**Comment by tomjaguarpaw** [comment]
> Yes -- good idea!

I agree. I don't have anything sensible to suggest about this but thank you for picking it up, it's important.

**Comment by visortelle** [comment]
Did you consider using services like Open Collective? https://opencollective.com/
Open Collective takes a 10% fee for their service but provides the ability to make a list of projects, budget goal per project, a list of contributors, contribution tiers, and more.

I found this as an example: https://opencollective.com/phpfoundation

**Comment by goldfirere** [comment]
> Did you consider 

No -- because I don't really have the capacity to take this on. I didn't consider anything in particular. Instead, I just wanted to start a conversation, perhaps leaving it for the ED to act on in due course, or an enterprising member of the community (cue @visortelle) to push forward on. If you think Open Collective has the right model for us, propose it! :)
